phpBB/adm/style/overall_header.html

```diff
@@ -118,7 +118,7 @@
 		<div id="acp">
 				<div id="content">
 					<div id="menu">
-						<p>{L_LOGGED_IN_AS}<br /><strong>{USERNAME}</strong> [&nbsp;<a href="{U_LOGOUT}">{L_LOGOUT}</a>&nbsp;][&nbsp;<a href="{U_ADM_LOGOUT}">{L_ADM_LOGOUT}</a>&nbsp;]&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p>
+						<p>{L_LOGGED_IN_AS} <strong>{USERNAME}</strong> [ <a href="{U_LOGOUT}">{L_LOGOUT}</a> ] [ <a href="{U_ADM_LOGOUT}">{L_ADM_LOGOUT}</a> ]</p>
```

Comment:
Hi,

Removing the br-tag and the nbsp between brackets `[ ]` could cause a display as bellow
```
You are logged in as: user1 [
 Logout ][ ACP Logout
 ]
```
IMO, only `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;` should be removed.